uui-components/src/inputs/RadioInput.tsx

```diff
@@ -52,10 +52,13 @@ export class RadioInput extends React.Component<RadioInputProps, any> {
                 <div className={ cx(uuiElement.radioInput, this.props.value && uuiMod.checked) }>
                     <input
                         type="radio"
-                        checked={ this.props.value }
                         disabled={ this.props.isReadonly || this.props.isDisabled }
+                        aria-disabled={ this.props.isReadonly || this.props.isDisabled }
```

Comment:
separate this.props.isReadonly and this.props.isDisabled and aria attrs. Make it in all input components which have readonly and disabled